try:
    print(x)
except:
    print("An error occured")